user-org-service/app/controllers/BaseController.java

```diff
@@ -39,190 +39,186 @@
  */
 public class BaseController extends Controller {
 
-    /**
-     * We injected HttpExecutionContext to decrease the response time of APIs.
-     */
-    @Inject
-    private HttpExecutionContext httpExecutionContext;
-    protected static Localizer localizerObject = Localizer.getInstance();
-
-    /**
-     * This is temporary method we use get dummyresponse to check APIs.
-     *
-     * @return string
-     */
-    public String getDummyResponse() {
-        startTrace("getDummyResponse");
-        String dummyResponse =
-                "{\"id\":\"api.user.200ok\",\"ver\":\"v1\",\"ts\":\"2019-01-17 16:53:26:286+0530\",\"params\":{\"resmsgid\":null,\"msgid\":\"8e27cbf5-e299-43b0-bca7-8347f7ejk5abcf\",\"err\":null,\"status\":\"success\",\"errmsg\":null},\"responseCode\":\"OK\",\"result\":{\"response\":{\"response\":\"SUCCESS\",\"errors\":[]}}}";
-        endTrace("getDummyResponse");
-        return dummyResponse;
-    }
-
-    public CompletionStage<Result> handelRequest() {
-        Http.RequestBody requestBody = request().body();
-        startTrace("handelRequest");
-        CompletableFuture<String> future = new CompletableFuture<>();
-        future.complete(getDummyResponse());
-        endTrace("handelRequest");
-        return future.thenApplyAsync(Results::ok, httpExecutionContext.current());
-    }
-
-    /**
-     * This method will return the current timestamp.
-     *
-     * @return long
-     */
-    public long getTimeStamp() {
-        return System.currentTimeMillis();
-    }
-
-    /**
-     * This method we used to print the logs of starting time of methods
-     *
-     * @param tag
-     */
-    public void startTrace(String tag) {
-        ProjectLogger.log(
-                String.format("%s:%s:started at %s", this.getClass().getSimpleName(), tag, getTimeStamp()),
-                LoggerEnum.DEBUG.name());
-    }
-
-    /**
-     * This method we used to print the logs of ending time of methods
-     *
-     * @param tag
-     */
-    public void endTrace(String tag) {
-        ProjectLogger.log(
-                String.format("%s:%s:ended at %s", this.getClass().getSimpleName(), tag, getTimeStamp()),
-                LoggerEnum.DEBUG.name());
+  /** We injected HttpExecutionContext to decrease the response time of APIs. */
+  @Inject private HttpExecutionContext httpExecutionContext;
+
+  protected static Localizer localizerObject = Localizer.getInstance();
+
+  /**
+   * This is temporary method we use get dummyresponse to check APIs.
+   *
+   * @return string
+   */
+  public String getDummyResponse() {
```

Comment:
Now Remove getDummyResponse () method, becuase , i think now this is not used.

Reply:
it is used by many controllers still

Reply:
ok